lib/bloc_record/associations.rb

```diff
@@ -44,8 +44,8 @@ def has_one(association)
         SELECT * FROM #{association_name}
         WHERE #{self.class.table}_id = #{self.id}
       SQL
-      
-      class_name = association_name.constantize
+
+      class_name = association_name.clasify.constantize
```

Comment:
This should be `classify` (you're missing an `s`). Have you actually used `has_one` or `has_many` in the address bloc?